jekyll/_cci2_ja/security-recommendations.md

```diff
@@ -108,4 +108,4 @@ EOF
 {: #see-also }
 {:.no_toc}
 
-[GitHub と Bitbucket のインテグレーション]({{ site.baseurl }}/ja/gh-bb-integration/)
+[GitHub と Bitbucket のインテグレーション]({{ site.baseurl }}/ja/github-integration/)
```

Comment:
@rosieyohannan I have done this replacement on my own. Can you please make sure this is accurate? thanks